client/src/javascript/i18n/translations/zh-Hant.json

```diff
@@ -264,7 +264,7 @@
   "torrents.list.no.torrents": "沒有任何種子可以顯示。",
   "torrents.list.drop": "將檔案拖曳到這裡來新增到 rTorrent。",
   "torrents.list.cannot.connect": "無法連線到 rTorrent。",
-  "torrent.list.peers": "{total} 中的 {connected}",
+  "torrent.list.peers": "{total} / {connected}",
```

Comment:
Outdated.

Reply:
There is a new translation in Crowdin.

Reply:
Crowdin will push updates automatically after a while.

Reply:
Ack. I will wait for the updates before merging.